src/ai/backend/manager/cli/__main__.py

```diff
@@ -170,6 +181,373 @@ def clear_history(cli_ctx: CLIContext, retention, vacuum_full) -> None:
         log.info('Cleaned up {:,} database records older than {:}.', deleted_count, expiration_date)
 
 
+@main.command()
+def configure() -> None:
```

Comment:
Since this function is quite long, how about moving all `mgr configure` stuffs into a separate module (`manager.cli.configure`)?